janus_core/log.py

```diff
@@ -11,7 +11,68 @@
   message: %(message)s
   trace: %(module)s
   line: %(lineno)d
-""".strip()
+"""
+
+
+class CustomFormatter(logging.Formatter):  # numpydoc ignore=PR02
```

Comment:
I don't think `CustomFormatter` gets across the purpose

Reply:
Is `YamlFormatter` better?